tests/foreman/cli/test_repository.py

```diff
@@ -304,6 +352,37 @@ def test_negative_create_with_name(self):
                 with self.assertRaises(CLIFactoryError):
                     self._make_repository({u'name': name})
 
+    @tier1
+    def test_negative_create_with_auth_url_forbidden_characters(self):
+        """Verify that repository URL cannot contain forbidden characters
+
+        @Feature: Repository
+
+        @Assert: Repository cannot be created
+        """
+
+        for cred in [creds for creds in valid_http_credentials()
```

Comment:
it is.
The characters *are* valid, but they need to be quoted in order to be placed into URL.
In this scenario i put them to url without quoting them

Please refer to the `valid_http_credentials()` docstring.